state/src/impls/test_helper.rs

```diff
@@ -16,8 +16,10 @@
 
 #![macro_use]
 
+#[allow(unused)]
```

Comment:
impolemented -> implemented
in the commit message

Reply:
Fixed